src/ViewInterface.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Starlit App.
+ *
+ * @copyright Copyright (c) 2016 Starweb AB
```

Comment:
```suggestion
 * @copyright Copyright (c) 2019 Starweb AB
```
Both here and in any new file